tests/lib/util/apply-disable-directives.js

```diff
@@ -343,19 +388,6 @@ describe("apply-disable-directives", () => {
                 []
             );
         });
-
-        it("keeps problems on the next line if there is an eslint-enable comment before the problem on the next line", () => {
```

Comment:
Sorry for the confusion... I dont think we should remove this test...
This behavior in this test does make sense....

Reply:
Really? I think the behavior is sort of strange -- this is the only place where the "block" directives and the "line" directives interact with each other. I think it makes more sense for them to be separate, and people can just use `/* eslint-disable */` rather than `// eslint-disable-line` if they only want to disable part of a line.

Reply:
We should probably decide about the correct behavior here. I think the version that's currently on this PR is the most reasonable choice (and at very least it's better than the version on master). If there is going to be a patch release tomorrow, then I think we should either decide this quickly or merge the PR as-is and decide it later.